Documentation/docs/migration_guides/itk_6_migration_guide.md

```diff
@@ -17,3 +17,12 @@ Remove support for ITKv4 interfaces
 `ITKV4_COMPATIBILITY` is no longer a supported option.  Previously this option
 was off by default.  Previously when enabled, the ITK API was modified to
 provide support for ITKV4 functions.
+
+Remove support for ITKv5
+-----------------------------------
+
+All contents of the Deprecated module were removed. This includes TreeContainer
+and related classes; atomic primitives, mutexes and related classes which are
+now part of C++ standard; specialized Vector filters - specialized versions are
```

Comment:
of the C++ standard library or standard C++ library ??

Reply:
atomics and mutexes are part of C++ since C++11. After we started requiring C++11 with ITKv5, we were able to deprecate ITK's variants.

Reply:
If the phrasing is unclear, please propose new phrasing.

Reply:
I think the original phrasing is OK as is.